insights/parsers/tests/test_abrt_status_bare.py

```diff
@@ -0,0 +1,18 @@
+from insights.parsers.abrt_status_bare import AbrtStatusBare
+from insights.tests import context_wrap
+
+OUTPUT = """
+420
+""".strip()
+
+OUTPUT_MULTILINE = """
+1997
+This line will never exist in real output, but it’s ignored.
```

Comment:
You have an error in this line with unicode char.  Take a look at the CI results.